tests/test_module_shutting.py

```diff
@@ -0,0 +1,69 @@
+import os
+import sys
+import unittest
+import subprocess
+import signal
+from os.path import dirname, abspath
+from time import time
+
+from core.messages import load_messages
+
+messages = load_messages().message_contents
+
+
+def run_container_in_sub_process(command):
+    is_network_traffic_capture_started = False
+    parent_directory = str(dirname(dirname(abspath(__file__))))
+    output = str()
+    expected_result = False
+    process = subprocess.Popen(command, stdout=subprocess.PIPE, stdin=subprocess.PIPE, shell=False,
+                               cwd=parent_directory)
+    start_time = time()
+    for c in iter(lambda: process.stdout.read(1), b""):
+        if time() - start_time > 300:
+            os.kill(process.pid, signal.SIGINT)
+            break
+        sys.stdout.buffer.write(c)
+        output += c.decode("utf-8")
+        if messages["network_traffic_capture_start"] in output and is_network_traffic_capture_started is False:
+            is_network_traffic_capture_started = True
+            os.kill(process.pid, signal.SIGINT)
+        elif is_network_traffic_capture_started is True and "finished." in output:
```

Comment:
should we use this statement on the top so that function is exited faster if case isn't satisfied?